### PR TITLE
Add option ShowOpenInDesktopOptionForSyncedFiles documentation to the Set-SPOTenant cmdlet

### PR DESCRIPTION
**Pull request by osarguel**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2012,6 +2013,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ShowOpenInDesktopOptionForSyncedFiles
+
+The ShowOpenInDesktopOptionForSyncedFiles setting (defaulted to false) enables opening a file locally in the desktop from the file item menu of SharePoint sites or OneDrive for Business Web, with the prerequisite that the file is being synced locally by the OneDrive client.
+
+The valid values are:
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2020]: >   [](http://example.com/codeflow?start=21&length=2)

Remove trailing spaces (here and a few places)

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2012,6 +2013,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ShowOpenInDesktopOptionForSyncedFiles
+
+The ShowOpenInDesktopOptionForSyncedFiles setting (defaulted to false) enables opening a file locally in the desktop from the file item menu of SharePoint sites or OneDrive for Business Web, with the prerequisite that the file is being synced locally by the OneDrive client.
+
+The valid values are:  
+
+- False (default) – Open In Desktop is disabled and the option is not shown in the file context menu.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2022]: >Open In Desktop [](http://example.com/codeflow?start=20&length=15)

should probably be "Open in desktop"

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -2012,6 +2013,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ShowOpenInDesktopOptionForSyncedFiles
+
+The ShowOpenInDesktopOptionForSyncedFiles setting (set to false by default) displays the “Open in desktop” option when users go to SharePoint or OneDrive on the web and open the shortcut menu for a file that they’re syncing with the OneDrive sync app.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2018]: retype these quotes so they're just `"`

here and other places.

Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2018]:
```suggestion
The ShowOpenInDesktopOptionForSyncedFiles setting (set to false by default) displays the "Open in desktop" option when users go to SharePoint or OneDrive on the web and open the shortcut menu for a file that they’re syncing with the OneDrive sync app.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -2012,6 +2013,26 @@ Accept pipeline input: False
...
+
+The valid values are:
+
+- False (default) – “Open in desktop” is disabled and not shown on the shortcut menu.
+- True – “Open in desktop” is enabled and the option to open synced files locally appears on the shortcut menu.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2023]:
```suggestion
- False (default) – "Open in desktop" is disabled and not shown on the shortcut menu.
- True – "Open in desktop" is enabled and the option to open synced files locally appears on the shortcut menu.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@osarguel, kindly commit the suggested changes. Many thanks.

**Review by samkabue — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by get-itips — COMMENTED** [review]
Checked in 16.0.22315.12000 and it is not there yet, is this going in the next release? thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by get-itips** [comment]
> Checked in 16.0.22315.12000 and it is not there yet, is this going in the next release? thank you

@osarguel

**Comment by osarguel** [comment]
> > Checked in 16.0.22315.12000 and it is not there yet, is this going in the next release? thank you
> 
> @osarguel

@samkabue  can you help me answering this?

**Comment by samkabue** [comment]
@osarguel this is not available in current versions, but should be in the release coming out next.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 5]
```diff
@@ -2013,6 +2014,25 @@ Accept pipeline input: False
...
+- True – "Open in desktop" is enabled and the option to open synced files locally appears on the shortcut menu.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: FalseAccept wildcard characters: False
```
Comment (tiburd) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2033]:
```suggestion
Accept pipeline input: FalseAccept
Wildcard characters: False
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 6]
```diff
@@ -2013,6 +2014,26 @@ Accept pipeline input: False
...
+- True – "Open in desktop" is enabled and the option to open synced files locally appears on the shortcut menu.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: FalseAccept
```
Comment (tiburd) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2033]:
```suggestion
Accept pipeline input: False
```

**Pull request merged**